tests/mechanic/launcher_test.py

```diff
@@ -147,7 +147,7 @@ def test_error_on_cluster_launch(self, sleep):
         cfg = config.Config()
         cfg.add(config.Scope.application, "client", "hosts", self.test_host)
         # Simulate that the client will raise an error upon startup
-        cfg.add(config.Scope.application, "client", "options", config_helper.ClientOptions('raise-error-on-info:true'))
+        cfg.add(config.Scope.application, "client", "options", opts.ClientOptions('raise-error-on-info:true'))
```

Comment:
Nit: Use double-quotes for string?

Reply:
Addressed in e04df54c7c682095b42e71ae4971cad82586eb73